CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 # Release Notes
 
-## 0.4.0 (unreleased)
+## 0.4.0 (13.07.2021)
+
+Release `0.4.0` brings many new major features to `weldx`
+
+### Highlights
+
+- [Quality Standards](https://weldx.readthedocs.io/en/latest/tutorials/quality_standards.html) \
+  Users can now create and integrate their own quality standards by defining new ASDF schema definitions and loading
+  them into weldx. It is possible to add new definitions or modify existing schemas to create your own flavour of the
+  weldx standard.
```

Comment:
Eventually we should note, that switching is not possible after the first validate operation (e.g. write op)?

Reply:
I suppose we can add that to the tutorial (and will update later anyway)